.github/workflows/aws_tfhe_integer_gpu_tests.yml

```diff
@@ -16,10 +16,17 @@ on:
   # Allows you to run this workflow manually from the Actions tab as an alternative.
   workflow_dispatch:
   pull_request:
+  push:
+    branches:
+      - main
```

Comment:
we run GPU test on push to main ?

Reply:
It's not mandatory as long we have a nightly run I guess.

Reply:
I think those we want to run on approve, we have the H100 tests for signed & unsigned operations that should run on push to the PRs

Reply:
So if I understand correctly, there is no need to trigger the workflow on push to `main`, only in PR on approval ?

Reply:
What I mean to say if I think I prefer to have them run on approve instead of nightly or push to main I think.